node_modules/oae-config/tests/test-config.js

```diff
@@ -1162,5 +1163,34 @@ describe('Configuration', function() {
             });
         });
 
+        /**
+         * Test that verifies that the guests tenant config can be updated just like any other tenant
+         */
+        it('verify the guests tenant config can be updated', function(callback) {
+            // Update the discussion visibility on the guests tenant
+            var config = {
+                'oae-discussions/visibility/discussion': 'private',
+                'oae-principals/recaptcha/enabled': false
+            };
+            ConfigTestUtil.updateConfigAndWait(globalAdminRestContext, 'guests', config, function(err) {
```

Comment:
The tests runner sets it to `guests`, seems a bit awkward to require the config in the tests. I wonder if we need to make the guests tenant alias configurable at all. It's not like we'd be exposing it to users anywhere and it might make a couple of things easier if we just hardcode the alias (through a constant). WDYT?